### DISCUSSION TITLE
Overwriting/Customizing layout templates

### DISCUSSION THREAD

**Discussion by michaelhume** [laravel/nova-issues#3890] [General]
Hi All, 

Is there a plan to implement a mechanism of overwriting layout templates? This has been mentioned in a few issues already #3885, #3889, and tangentially for the logo. In V3 there was some control over this via the layout.blade file.

Specifically, I'd like to do small things like: 
 * make app bar sticky
 * fix the sidebar position and have it scrollable 
 * better control over logo & footer

Thanks!

**Comment by elizdlt202** [comment]
@michaelhume Hi, so in your install of version 4 were those views missing as well? Only the layout.blade was included in v4

**Reply by michaelhume** [reply]
Yes, the FE is all Inertia now so I don't know if "missing" is the right word. It just works differently now and even the smallest customizations are non-trivial, if even possible at all without digging into vendor files. 

Support for [themes](https://nova.laravel.com/docs/3.0/customization/themes.html) seems gone as well.

**Reply by elizdlt202** [reply]
Thanks so much for answering. I see now what you're referring to !

**Comment by smartens80** [comment]
Yes, this is definitely holding us back on the upgrade from v3 -> v4

We need to be able to make basic layout changes as well as theme / css changes easily like we could in v3.
